### PR TITLE
Add check for empty/null strings. #337

### PR DESCRIPTION
**Pull request by IvanSheyenko**

### CODE REVIEW

File: src/SIM.Tool.Windows/UserControls/Install/Instance9Details.xaml.cs [hunk 1]
```diff
@@ -108,6 +108,11 @@ public bool OnMovingNext(WizardArgs wizardArgs)
       args.Product = product;
 
       SolrDefinition solr = this.Solrs.SelectedItem as SolrDefinition;
+      if (solr == null)
+      {
+        WindowHelper.ShowMessage("Please provide solr");
```
Comment (ViacheslavAfanasyev) [inline src/SIM.Tool.Windows/UserControls/Install/Instance9Details.xaml.cs]: There is no '.' symbol at the end of the string-like with another your window.

File: src/SIM.Tool.Windows/UserControls/Install/Instance9Details.xaml.cs [hunk 2]
```diff
@@ -108,6 +108,11 @@ public bool OnMovingNext(WizardArgs wizardArgs)
       args.Product = product;
```
Comment (ViacheslavAfanasyev) [inline src/SIM.Tool.Windows/UserControls/Install/Instance9Details.xaml.cs]: You haven't covered the case when the site name field is empty.

File: src/SIM.Tool.Windows/UserControls/Install/Instance9Details.xaml.cs (hunk 1 above)
Comment (IvanSheyenko) [reply src/SIM.Tool.Windows/UserControls/Install/Instance9Details.xaml.cs]: fixed

File: src/SIM.Tool.Windows/UserControls/Install/Instance9Details.xaml.cs (hunk 2 above)
Comment (IvanSheyenko) [reply src/SIM.Tool.Windows/UserControls/Install/Instance9Details.xaml.cs]: There is an error message for this installation does not fail

**Pull request merged**